### DISCUSSION TITLE
#20. VATZ biweekly meeting at 2022-10-26 08:00:00 -06:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#274] [General]
### 1. Overall
<img width="1213" alt="image" src="https://user-images.githubusercontent.com/6308023/195475343-0387415a-a331-4637-b50f-965975a4327e.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
18|  1/18 | 5
19|  8/22 | 36

---

- Finished
   - #259
   - #262 
   - #256 
   - #226 
   - #246 
   - #174 
   - #220 
    
- Move to Sprint 19
   - #85 
   - #87 
   - #119 
   - #121 
   - #122 
   - #146 
   - #175 
   - #187 
   - #197 
   - #202 
   - #228 
   - #238 
   - #242 
   - #261  
     
---

**Comment by rootwarp** [comment]
# 준교
- #275 
- #277 
- #278 
- #281

**Comment by xellos00** [comment]
# 동요
- #180 
- #261 
- #282 
- #285

**Reply by xellos00** [reply]
from @rootwarp 
good to have panic in codes

**Reply by xellos00** [reply]
@heejin-github 
Official-Plugin will be done by Next Sprint.

**Reply by xellos00** [reply]
@hqueue 
- minimize, No outdated, No wrong Info
  - How to start
  - Command list (@rootwarp)
  - Config

**Comment by heejin-github** [comment]
## 희진
- #236 
  - #288

**Comment by kim201212** [comment]
# 형기

- https://github.com/dsrvlabs/vatz/issues/236
   - https://github.com/dsrvlabs/vatz/issues/266
   - https://github.com/dsrvlabs/vatz/issues/268
   - https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/19

**Comment by skonhwang** [comment]
# 승곤
- https://github.com/dsrvlabs/vatz/issues/236
    - https://github.com/dsrvlabs/vatz/issues/267

**Comment by gnongs** [comment]
## 진홍
- https://github.com/dsrvlabs/vatz/issues/280

**Comment by xellos00** [comment]
# 황제
- no issue

**Comment by xellos00** [comment]
# 영빈
